blueocean-web/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+    "presets": [
+        "es2015", "react", "stage-0"
+    ],
+    "plugins": [
+        "transform-decorators-legacy"
+    ]
+}
```

Comment:
I think in practice there won't be many projects that use our Maven multi-module approach to organization. Beyond that, please see points 1 and 2 from this comment here:  https://github.com/jenkinsci/js-builder/pull/4#issuecomment-239001170

I think trying to implement our own behavior that's close to how babel works OOTB but slightly different is a bad path to go down, and I'd hate to see us have to add another package.json in this project that needs to be managed.